syntax = "proto3";

package zonk;

service ZonkService {
  // Method for delect dice
  rpc GetSelectedDices(SelectedDicesRequest) returns (SelectedDicesResponse);

  // Method for ask about continue game
  rpc GetContinuationDecision(ContinuationDecisionRequest) returns (ContinuationDecisionResponse);
}

// Request params for GetSelectedDices 
message SelectedDicesRequest {
  string game_id = 1;
  repeated int32 dices = 2;
}

// Response with selected dices
message SelectedDicesResponse {
  repeated int32 dices = 1;
}

// Request params for GetContinuationDecision (может быть пустым)
message ContinuationDecisionRequest {
 string game_id = 1;
}

// Response with continue game
message ContinuationDecisionResponse {
  bool continue_game = 1;
}